### PR TITLE
Update linux-install-manually.md

### LINKED ISSUE #12660: On Debian 12, the "Configure the Linux software repository" step gives an error.
[Enter feedback here]
Areas that need fixing:
The following command for "Ubuntu and Debian systems" in "Configure the Linux software repository"

・Update the repository metadata:
   sudo apt-get update
  

Reasons to fix:
In Debian 12, apt-key is deprecated (obsolete), so public key authentication is not possible with the current documentation procedures.
As per the documentation, when I run the "sudo apt-get update" command, I get the following error:
━━━━━━━━━━━━━━━━━━━━
W: GPG error: https://packages.microsoft.com/debian/12/prod bookworm InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY EB3E94ADBE1229CF
E: The repository 'https://packages.microsoft.com/debian/12/prod bookworm InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
━━━━━━━━━━━━━━━━━━━━

To avoid the above error, the Microsoft GPG public key "microsoft.gpg" installed with the following command must be placed as "/usr/share/keyrings/microsoft-prod.gpg".
By doing this, the "sudo apt-get update" command on Debian 12 will succeed.

Command:
curl -sSL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/microsoft.gpg > /dev/null

Please add to the documentation that Debian 12 requires special work.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: bab0b894-0199-7515-b060-a6724e2c9b9e
* Version Independent ID: 988ff83c-c4b5-d2f3-023e-2d8dd191b7ae
* Content: [Deploy Microsoft Defender for Endpoint on Linux manually](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/linux-install-manually?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-endpoint/linux-install-manually.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/linux-install-manually.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the location of GPG Public key

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12660

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Comment by narkuroki** [comment]
@yogkumgit 
@msbemba 
Unfortunately, the command you're trying to fix fails on Debian 11 and below.
What I wanted in this request was to write instructions for both pre-Debian 11 and he for Debian 12.
It seems that my request was not conveyed correctly.
Could you please fix it again as below?

・Contents currently published in the document
━━━━━━━━━━━━━━━━━━━━
・Install the Microsoft GPG public key:
curl -sSL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/microsoft.gpg > /dev/null
━━━━━━━━━━━━━━━━━━━━

・What you are currently trying to modify with "https://github.com/MicrosoftDocs/microsoft-365-docs/pull/12695"
━━━━━━━━━━━━━━━━━━━━
・Install the Microsoft GPG public key:
curl -sSL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor | sudo tee /usr/share/keyrings/microsoft-prod.gpg > /dev/null
━━━━━━━━━━━━━━━━━━━━

・Real content that you want to correct
━━━━━━━━━━━━━━━━━━━━
・Install the Microsoft GPG public key:

For Debian 11 and below, run the following command.
curl -sSL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/microsoft.gpg > /dev/null

For Debian 12 and above, run the following command.
curl -sSL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor | sudo tee /usr/share/keyrings/microsoft-prod.gpg > /dev/null
━━━━━━━━━━━━━━━━━━━━

File: microsoft-365/security/defender-endpoint/linux-install-manually.md [hunk 1]
```diff
@@ -201,10 +201,16 @@ In order to preview new features and provide early feedback, it is recommended t
 
 - Install the Microsoft GPG public key:
 
+For Debian 11 and below, run the following command.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/linux-install-manually.md]:
```suggestion
For Debian 11 and earlier, run the following command.
```

File: microsoft-365/security/defender-endpoint/linux-install-manually.md [hunk 2]
```diff
@@ -201,10 +201,16 @@ In order to preview new features and provide early feedback, it is recommended t
...
+
   ```bash
-  curl -sSL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/microsoft.gpg > /dev/null
+  curl -sSL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor | sudo tee /usr/share/keyrings/microsoft-prod.gpg  > /dev/null
   ```
+For Debian 12 and above, run the following command.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/linux-install-manually.md]:
```suggestion
For Debian 12 and later, run the following command.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by narkuroki** [comment]
@JohanFreelancer9 
The current content is the same command for both Debian 11 and earlier and Debian 12 and later.
image

Correctly, I would like it to be written as follows.

For Debian 11 and earlier, run the following command.
curl -sSL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/microsoft.gpg > /dev/null

For Debian 12 and later, run the following command.
curl -sSL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor | sudo tee /usr/share/keyrings/microsoft-prod.gpg > /dev/null

File: microsoft-365/security/defender-endpoint/linux-install-manually.md [hunk 3]
```diff
@@ -201,10 +201,16 @@ In order to preview new features and provide early feedback, it is recommended t
 
 - Install the Microsoft GPG public key:
 
+For Debian 11 and earlier, run the following command.
+
   ```bash
-  curl -sSL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/microsoft.gpg > /dev/null
+  curl -sSL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor | sudo tee /usr/share/keyrings/microsoft-prod.gpg  > /dev/null
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/linux-install-manually.md]:
```suggestion
  curl -sSL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/microsoft.gpg > /dev/null
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba as per @narkuroki's request.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks.

**Comment by yogkumgit** [comment]
@Dansimp Gentle reminder.

**Comment by narkuroki** [comment]
@yogkumgit 
@Dansimp 
Is it possible to correct the documentation regarding this matter?
Thank you for your cooperation.

**Pull request merged**